.flowconfig

```diff
@@ -0,0 +1,20 @@
+[ignore]
```

Comment:
Unfortunately, yeah :/ This is how Flow knows which code is yours and which code is library stuff.